README.md

```diff
@@ -20,6 +20,7 @@ Fluttium, the user flow testing tool for Flutter.
 
 | Package                                                                                        | Pub                                                                                                       |
 | ---------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+| [fluttium](https://github.com/wolfenrain/fluttium/tree/main/packages/fluttium) | [![pub package](https://img.shields.io/pub/v/fluttium.svg)](https://pub.dev/packages/fluttium)    |
```

Comment:
What about `fluttium_driver`, `flutter_interfaces`, and `fluttium_protocol`?

Reply:
Good catch, updated it!